collectors/macos.plugin/multi_metadata.yaml

```diff
@@ -42,14 +48,208 @@ modules:
         list: []
       configuration:
         file:
-          name: ""
-          description: ""
+          name: "netdata.conf"
+          section_name: |
+            - `[plugin:macos:sysctl"]` - Enable or disable monitoring for network, swap, loadavg, and boot time.
+            - `[plugin:macos:mach_smi]` - Enable or disable monitoring for CPU and Virtual memory.
+            - `[plugin:macos:iokit]` - Enable or disable monitoring for storage device.
+          description: "The netdata main configuration file."
         options:
           description: ""
           folding:
-            title: ""
+            title: "Config options"
             enabled: true
-          list: []
+          list:
+            - name: enable load average
+              section_name: plugin:macos:sysctl
```

Comment:
This attribute is not in the schema, it will do nothing at the moment, I think we need to think how to communicate this collector better to the user..

Possibly mentioning it in the description, or add a prefix to the option name

Reply:
This was an @ilyam8 's suggestion in another PR, and I began to use it, probably we need to add something like this, because different plugins that have options inside `netdata.conf` will have sections like this.

Reply:
What was my suggestion? To use non-existent keys?

Reply:
I messed up `section_name` for different keywords, sorry for this.